src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -168,9 +168,28 @@ private void assertCommandBehavior(String inputCommand, String expectedMessage,
         //Confirm the state of data (saved and in-memory) is as expected
         assertEquals(expectedTaskList, model.getTaskMaster());
         assertEquals(expectedTaskList, latestSavedTaskList);
+        
     }
+    
+    private void assertUndoRedoAble(String expectedMessage,
```

Comment:
As undo/redo sometimes screw up with strange reason, add this assertion to ensure a command can be undo and redo.